data/transportation/roads-system/index.html

```diff
@@ -209,7 +195,7 @@ <h3 id="TenthMileReferencePoints">
     <p>The native spatial reference for this dataset is UTM Zone 12N, NAD83 (0.01 meter coordinate precision). This dataset was assembled by UDOT for general purpose cartography.</p>
     <h5>Related Resources</h5>
```

Comment:
Remove this section if there are no related items. Otherwise this all LGTM. Let me know when you want a merge.

Reply:
awesome, thanks for checking it.  you can merge anytime after 10am tomorrow (sept 13th).  i can remind you again, then.  yeah, we still serve the tenth-mile points for them - they update about every 2 to 3 months with me during a road update.